scripts/prepare_db.py

```diff
@@ -46,17 +47,17 @@
 
 #%%
 # def summary_h2(n, t):
-t = 24

```

Comment:
This and the following line are neither in main nor in any other function, there is no indent?